articles/azure-app-configuration/rest-api.md

```diff
@@ -5,14 +5,14 @@ author: maud-lv
 ms.author: malev
 ms.service: azure-app-configuration
 ms.topic: reference
-ms.date: 08/17/2020
+ms.date: 11/28/2022
 ---
 
 # Azure App Configuration REST API
```

Comment:
Seems that there is a desire to distinguish data plane here. Should the header get an update as well?

Reply:
Good idea. Updated.